src/vpn-worker.ts

```diff
@@ -205,14 +207,23 @@ const worker = async (instanceId: number, serviceId: number) => {
 		logger.info(
 			`connection established with client_id=${clientId} uuid=${data.username}`,
 		);
+		// We need to flatten the uuid because otherwise it's a sliced string and keeps the
+		// original, rather large, string in memory forever
+		const uuid = flatstr(data.common_name);
 		clientCache[clientId] = {
-			// We need to flatten the uuid because otherwise it's a sliced string and keeps the
-			// original, rather large, string in memory forever
-			uuid: flatstr(data.common_name),
+			uuid,
 			bytes_received: 0,
 			bytes_sent: 0,
 			ts: process.hrtime()[0],
 		};
+
+		request.post({
```

Comment:
Should we also have a `.catch` (+ report ?) so that the process doesn't crash if this request fails?